### PR TITLE
fix buffer underruns on epc2

### PR DESCRIPTION
**Pull request by hhoegelo**
* remove unnecessary setting of FRAMES_PER_PERIOD to the value it has anyway
* reload snd_hda_intel driver if it exists (should do nothing on old epc)

### CODE REVIEW

**Review by AntonSchmied — APPROVED** [review]
Looks freaky, but okay! I will test it on the older NUCs and give an  update

**Review by schlenkibus — APPROVED** [review]

File: projects/epc/audio-engine/system/audio-engine.service.in [hunk 1]
```diff
@@ -13,9 +13,11 @@ ExecStartPre=/usr/bin/bash -c "while [ ! -f /proc/asound/EMPHASE/id ]; do sleep
 # WAIT FOR AUDIO DEVICE
 ExecStartPre=/usr/bin/bash -c "while [ ! -f /proc/asound/PCH/id ]; do sleep 0.1; done"
 
+# CONFIGURE AUDIO DEVICE DRIVER
+ExecStartPre=-/usr/bin/bash -c "[ -e /sys/module/snd_hda_intel/parameters/bdl_pos_adj ] && rmmod snd_hda_intel && modprobe snd_hda_intel bdl_pos_adj=1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1"
```
Comment (AntonSchmied) [inline projects/epc/audio-engine/system/audio-engine.service.in:17]: Question: what does the '-'  do here -> 'ExecStartPre= - /usr/...'

Comment (schlenkibus) [reply projects/epc/audio-engine/system/audio-engine.service.in:17]: Upon trying to run the lines on a already running EPC 1, I get following results back:
```zsh: no such file or directory: -/usr/bin/bash```


dropping the '-' leads to following output: 

```rmmod: ERROR: Module snd_hda_intel is in use```

Comment (AntonSchmied) [reply projects/epc/audio-engine/system/audio-engine.service.in:17]: same case on ePC2

Comment (AntonSchmied) [reply projects/epc/audio-engine/system/audio-engine.service.in:17]: Answer: "There in docs for ExecStartPre=, you'll find it explained that the leading "-" is used to note that failure is tolerated for these commands. In this case, it's tolerated if /run/thing already exists."

**Review by schlenkibus — APPROVED** [review]

**Pull request merged**